dbsettings/values.py

```diff
@@ -306,20 +304,13 @@ def get_db_prep_save(self, value, oldvalue=None):
 
         hashed_name = md5(str(time.time()).encode()).hexdigest() + value.name[-4:]
         image_path = pjoin(self._upload_to, hashed_name)
-        dest_name = pjoin(settings.MEDIA_ROOT, image_path)
-        directory = pjoin(settings.MEDIA_ROOT, self._upload_to)
 
-        if not os.path.exists(directory):
-            os.makedirs(directory)
-        with open(dest_name, 'wb+') as dest_file:
-            for chunk in value.chunks():
-                dest_file.write(chunk)
+        default_storage.save(image_path, value.file)
```

Comment:
Documentation says `[...] the storage system may modify the filename as necessary to get a unique name. The actual name of the stored file will be returned.`
Please get the return value from `default_storage.save` and use it as new `image_path`

Reply:
You're absolutely right, I updated the line.